libnetwork/osl/interface_linux.go

```diff
@@ -1,7 +1,9 @@
 package osl
 
 import (
+	"bytes"
 	"context"
+	stderrors "errors"
```

Comment:
It looks like the only place `pkg/errors` is used in this file is in `Namespace.setSysctls` (added in 0071832226b42dbb12bc01bf1c668dca891b2d33) and that function already mixes `fmt.Errorf` with `errors.Wrap`. Perhaps we're good to change those to use `errors` as well so that we don't have to use the special alias; https://github.com/moby/moby/blob/9e3e42971a887e1ef20ac4eb7c6e542cba547c34/libnetwork/osl/interface_linux.go#L462-L500



Alternatively, we could use the `internal/multierror`  package, which I think was created for better formatting, but not sure if we need to have that.